test/cmdlineTests.sh

```diff
@@ -240,35 +240,42 @@ printTask "Running general commandline tests..."
     cd "$REPO_ROOT"/test/cmdlineTests/
     for tdir in */
     do
-        if [ -e "${tdir}/input.json" ]
+        printTask " - ${tdir}"
+
+        inputFiles="$(find "${tdir}" -name 'input.*' -type f -exec printf "%s\n" "{}" \;)"
+        inputCount="$(echo "${inputFiles}" | wc -l)"
+        if (( ${inputCount} == 0 ))
+        then
+            printError "No input files found."
+            exit 1
+        fi
+
+        if (( ${inputCount} > 1 ))
         then
+            printError "Ambiguous input. Found input files in multiple formats:"
+            echo -e "${inputFiles}"
+            exit 1
+        fi
+
+        # Use printf to get rid of the trailing newline
+        inputFile=$(printf "%s" "${inputFiles}")
+
+        if [ "${inputFile}" = "${tdir}input.json" ]
+        then
+            stdin="${inputFile}"
             inputFile=""
-            stdin="${tdir}/input.json"
-            stdout="$(cat ${tdir}/output.json 2>/dev/null || true)"
-            stdoutExpectationFile="${tdir}/output.json"
-            args="--standard-json "$(cat ${tdir}/args 2>/dev/null || true)
+            stdout="$(cat ${tdir}output.json 2>/dev/null || true)"
+            stdoutExpectationFile="${tdir}output.json"
+            args="--standard-json "$(cat ${tdir}args 2>/dev/null || true)
```

Comment:
Sure that `tdir` always contains `/` at the end?

Reply:
Yeah. There was a mix of stuff like `${tdir}output` and `${tdir}/args` before my change. Now it's just more consistent.

It's defined as `for tdir in */` and I also saw the values when debugging this.

Reply:
Ok